eventually/amazons3/views.py

```diff
@@ -81,11 +82,11 @@ def delete(self, request):
         """
 
         if request.content_type != 'application/json':
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
 
         image_to_delete = json.loads(request.body)
         if not image_to_delete.get("image_key"):
-            return HttpResponse(status=400)
+            return RESPONSE_400_INVALID_DATA
 
         delete_obj = {"Key": image_to_delete.get("image_key")}
         BUCKET.delete_objects(Delete={"Objects":[delete_obj]})
```

Comment:
http://boto3.readthedocs.io/en/latest/reference/services/s3.html#S3.Client.delete_objects

Reply:
Verified yesterday, rewritten the code via Client() but it hasn't given much difference.